### PR TITLE
android: fix promise not called anymore after the first try

### PR DESCRIPTION
**Pull request by NicholasBertazzonAga**
Related to #67

### CODE REVIEW

File: android/src/main/java/com/smarkets/paypal/RNPaypalModule.java [hunk 1]
```diff
@@ -55,8 +57,6 @@ public void requestOneTimePayment(
 
     if (options.hasKey("currency"))
         request.currencyCode(options.getString("currency"));
-    if (options.hasKey("displayName"))
```
Comment (nzankich) [inline android/src/main/java/com/smarkets/paypal/RNPaypalModule.java:58]: Why did you remove display name here?

Comment (NicholasBertazzonAga) [reply android/src/main/java/com/smarkets/paypal/RNPaypalModule.java:58]: Oh sorry I didn't mean to

**Comment by NicholasBertazzonAga** [comment]
@nzankich there you go restored, sorry about it

**Comment by nzankich** [comment]
@NicholasBertazzonAga no problem!  thanks for fixing this

I've approved another PR since yours that includes the same promise pattern.  Guessing I should update that one as well? (I can do it though)
